packages/algob/test/internal/core/params/argument-types.ts

```diff
@@ -232,19 +232,21 @@ describe("argumentTypes", () => {
       );
     });
 
-    it("Should throw if the file isn't readable", async function () {
-      if (os.type() === "Windows_NT") {
+    it("Should throw if the file isn't readable", function () {
+      var isRoot = process.getuid && process.getuid() === 0;
+      if (os.type() === "Windows_NT" || isRoot) {
         this.skip();
       }
-      await fsExtra.createFile("A");
-      await fsExtra.chmod("A", 0);
+
+      fsExtra.createFileSync("A");
+      fsExtra.chmodSync("A", 0o000); // assign no permission [read-execute-write: 000]
```

Comment:
what is the benefit of using 0o000 instead of 0 ?

Reply:
also why are we removing await. is it possible that may result in error in some systems ?

Reply:
`0o000` is more readable because you show all flags. - each digit is rights for user, group and anyone else receptively.